earth_enterprise/src/fusion/portableglobe/cutter/cgi-bin/globe_cutter_app.py

```diff
@@ -305,8 +305,7 @@ def CreateInfoFile(self):
     """Create globe info file."""
     fp = open(self.info_file, "w")
     fp.write("Portable Globe\n")
-    fp.write("Copyright 2017 Google Inc.
-              Licensed under the Apache License, Version 2.0.\n")
+    fp.write("Copyright 2017 Google Inc.\nLicensed under the Apache License, Version 2.0.\n");
```

Comment:
I've seen plenty of code where the Python interpreter seems to ignore people's semicolons fine, but it's not Python.

You could also use single multi-line string instead of a series of `write`s if you prefer.  You just need to triple the quotes.